Source/Application.js

```diff
@@ -37,19 +31,16 @@ export class Application extends WrappedObject {
    * @return A new Application object.
    */
   constructor(context) {
-    super(context)
+    log('using `api()` is deprecated. Use `api.X` directly')
```

Comment:
this message needs to be consistent with whatever we end up calling the global

Reply:
I'm not expecting people to use the function form outside `context.api()` so it will probably always be `api` here